### ISSUE TITLE
Incompatibility of Eclipse plugins STS Quicksearch, sonar-ide and m2e

### ISSUE THREAD

**Issue by may-bee** [oasp/oasp4j-ide#13]
I ran into an incompatibility problem after upgrading my Luna plug-ins to the latest version.
Spring Tool Suite 3.6 is incompatible with SonarQube Plug-ins of version 3.4. The problem is an non-standardized artifact of slf4j, which is imported by SonarQube. After uninstalling SonarQube, m2e does ist job again.

For more information see [here](http://forum.spring.io/forum/spring-projects/springsource-tool-suite/749864-sts-ggts-3-6-0-m1-released)

_Just a hint for further central ide upgrades_

**Comment by may-bee** [comment]
I unpacked the current 1.3.0 release and installed a new snapshot version of CobiGen as well as Eclipse Quicksearch.

The current configuration looks like:
```
Eclipse Quicksearch 3.6.3.21411271002-RELEASE
m2e 1.5.0.20140606-0033
SonarQube 3.4.0.20140404-0949-RELEASE
```

No Problems at all.

**Comment by hohwille** [comment]
SonarQube 3.4.0.20140404-0949-RELEASE still conflicts with m2e buildhelper in this constellation.

**Comment by hohwille** [comment]
If nothing changes, we have to decide between SonarQube and QuickSearch in the next release.

**Comment by lgoerlach** [comment]
Fixed in release candidate 1.4.0
